releasenotes.rb

```diff
@@ -48,6 +48,16 @@ class ReleaseNotes < Thor
 
 ## Non-Customer Facing Changes
 
+## Special Deploy Requirements
+
+<% if commits.select(&:special_deploy_requirements).empty? %>
```

Comment:
ahh damn i didn't notice that! ok .. i fear that if they are at the bottom then people won't notice them .. but they are probably the most important part of the deploy ( for developers ). I can look into pulling the requirements but it might be a little hard to distinguish between what a requirement is and what normal text is.